Documentation/compatibility/wcf-accessbility-high-contrast-issue-in-search-in-combo-box- in-svcTraceViewer-tool.md

```diff
@@ -0,0 +1,34 @@
+
+## [svcTraceReviewer Search in combo box high contrast change]
+
+### Scope
+Edge
+
+### Version Introduced
+4.8
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In svcTraceViewer tool, the open button in Search in combo box in "Microsoft Service Trace Viewer" window was not displayed in black color in high contrast black mode (or white color in high contrast white mode). The issue was fixed in 4.7.2 .NET Framework SDK. However, due to .NET Framework SDK backward compatible requirement, it can't target 4.7.2 .NET Framework version, the fix was not visible to customer by default. An additional  fix is made in 4.8 to add the following configuration switch to svcTraceViewer.exe.config file
```

Comment:
It says "The issue was fixed in 4.7.2 .NET Framework SDK." Shouldn't it just be "The issue was fixed in the .NET 4.7.2 Framework"? I don't think we made any changes to the 4.7.2 SDK for this. Technically if they add the config setting themselves this issue would be fixed on any version of the svcTraceViewer tool as long as they have the 4.7.2 framework installed.